src/app/modules/products/components/product-detail/product-detail.component.html

```diff
@@ -13,18 +13,16 @@ <h1 class="product-detail-content-title">{{ product.name }}</h1>
                 </div>
                 <form [formGroup]="optionsForm" class="product-detail-selection-form">
```

Comment:
there is a form tag here and one below, we can only use one and nest everything within it to keep things semantically clean.

(minor adjustment)

Reply:
done